### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by g33kidd**
Was this a typo?

### CODE REVIEW

File: docs/api-context.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ char* source_map_file;
 char* source_map_root;
 ```
 ```C
-// Custom functions that can be called from sccs code
+// Custom functions that can be called from sass code
```
Comment (xzyfer) [inline docs/api-context.md]: Should be `Sass` (not the uppercase S).

**Comment by am11** [comment]
Same typo is present in `docs/api-context-internal.md` and `src/sass_context.hpp`.

**Comment by mgreter** [comment]
Anyone up for the task to clean this up?

**Comment by xzyfer** [comment]
This isn't ready. Feels stalled.

**Pull request closed without merging**